salalib/mgraph.cpp

```diff
@@ -419,40 +419,40 @@ bool MetaGraph::makeShape(const Line& line)
    return (map.makeLineShape(line,true) != -1);
 }
 
-bool MetaGraph::polyBegin(const Line& line)
+int MetaGraph::polyBegin(const Line& line)
 {
    if (!isEditableMap()) {
       return false;

```

Comment:
That return value needs to be changed - returning `false` for an integer works but is bad form. I'd suggest `-1`.